test/tests/spack_tests.py

```diff
@@ -0,0 +1,63 @@
+import io
+from pathlib import Path
+import re
+
+from pavilion import arguments
+from pavilion import commands
+from pavilion import plugins
+from pavilion import test_run
+from pavilion.unittest import PavTestCase
+from pavilion.plugins.commands.status import get_tests
+
+class SpackTests(PavTestCase):
+
+    def setUp(self):
+
+        self.pav_cfg['spack_path'] = Path('~/a/fake/path')
+        self.working_dir = self.pav_cfg['working_dir']
+        plugins.initialize_plugins(self.pav_cfg)
+
+    def tearDown(self):
+
+        plugins._reset_plugins()
+
+    def test_spack_build(self):
+        """Test to ensure that a test is built correctly."""
```

Comment:
You should condition this test on pav_cfg.spack_path being set and existing.
The unittest module has function decorators for that (the slurm tests use it)